server/testutils/mmcontainer/testscontainer.go

```diff
@@ -25,7 +24,7 @@ const (
 	defaultPassword        = "admin"
 	defaultTeamName        = "test"
 	defaultTeamDisplayName = "Test"
-	defaultMattermostImage = "mattermost/mattermost-enterprise-edition"
+	defaultMattermostImage = "mattermost/mattermost-enterprise-edition:release-9.6"
```

Comment:
Should we avoid hard-coding this, lest we forget and keep testing on what will be an old version?

Reply:
That is going to happen, at least locally, anyway, because if you don't specify the version is going to use the one that you have already in your cache. Also, having it explicitly ensure that the plugin works in previous versions, if it fails, you detect that you need to bump the version of mattermost in the requirements of the plugin.json.